src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Parcela.kt

```diff
@@ -21,16 +23,18 @@ class Parcela(val ancho: Int, val largo: Int, val horasSolPorDia: Int) {
 }
 
 class Agricultora(val parcelas: MutableList<Parcela>) {
```

Comment:
`parcelas` no debería ser mutable
```suggestion
class Agricultora(val parcelas: listOf<Parcela>) {
```

Reply:
No entiendo porque. (Si entiendo la diferencia de una MutableList y una listOf)

Reply:
Porque el requerimiento lo dice:

> Para este ejercicio, diremos que la tierra no puede ser ni vendida ni comprada, y por lo tanto cuando se crea una agricultora se configura con todas las parcelas que tiene.

Reply:
Agricultora.parcelas -> debería ser inmutable, porque en el enunciado dice que no se pueden comprar ni vender.

Reply:
Me conteste solo. Gracias!